dashboard-for-desktop/articles/dashboard-designer/designing-dashboard-items/grid/conditional-formatting.md

```diff
@@ -4,42 +4,65 @@ author: Natalia Kazakova
+	* [Bar Gradient Ranges](../../appearance-customization/conditional-formatting/bar-gradient-ranges.md) 
+* string 
+	* [Value](../../appearance-customization/conditional-formatting/value.md) (with a condition type set to _Equal To_, _Not Equal To_ or _Text that Contains_)
+	* [Expression](../../appearance-customization/conditional-formatting/expression.md)
+* date-time
+	* [Value](../../appearance-customization/conditional-formatting/value.md)
+	* [A Date Occurring](../../appearance-customization/conditional-formatting/value.md) (for dimensions with a continuous date-time group interval)
+	* [Expression](../../appearance-customization/conditional-formatting/expression.md)
+	* [Icon Ranges](../../appearance-customization/conditional-formatting/icon-ranges.md)
+	* [Color Ranges](../../appearance-customization/conditional-formatting/color-ranges.md)
+	* [Gradient Ranges](../../appearance-customization/conditional-formatting/gradient-ranges.md)
+	* [Bar](../../appearance-customization/conditional-formatting/bar.md) 
+	* [Bar Color Ranges](../../appearance-customization/conditional-formatting/bar-color-ranges.md) 
+	* [Bar Gradient Ranges](../../appearance-customization/conditional-formatting/bar-gradient-ranges.md) 
+
+## Create and Edit a Format Rule
+
+You can create and edit format rules in the following ways:
+
+* Click the **Edit Rules** button on the **Home** ribbon tab. 
+
+* Click the measure/dimension menu button in the Data Item's pane and select **Add Format Rule**/**Edit Rules**. 
+
+Refer to the following topic for information on how to create and edit format rules: [Conditional Formatting Common](../../appearance-customization/conditional-formatting.md).
+
+## Grid-Specific Format Condition Settings
+
+Specify appearance settings and set the condition's value for the format rule. Available settings depend on the selected format rule. The image below displays the **Greater Than** dialog, which corresponds to the [Value](../../appearance-customization/conditional-formatting/value.md) format condition for a Grid dashboard item. The format condition applies to the _ExtendedPrice_ data item in the **Apply to** drop-down list.
 
 ![GreaterThanDialog](../../../../images/img118555.png)
 
 The **Apply to row** check box allows you to specify whether to apply the formatting to the entire grid row.
 
-## <a name="edit-a-format-rule"/>Edit a Format Rule
-To edit format rules for the current Grid dashboard item, use the following options.
-* Click the **Edit Rules** button in the **Home** ribbon tab or use corresponding item in the Grid context menu.
-* Click the [menu button](../../ui-elements/data-items-pane.md) for the required data item and select **Edit Rules**. As an alternative, right-click the column header corresponding to the required data item and select **Edit Rules**.
+You can configure and customize current format condition appearance settings.
```

Comment:
Does this sentence relate to the two items below? If so, can we update the text so that it reflects this? For example, do you mean:

`You can do the following to configure and customize format condition appearance settings:`